include/z64scene.h

```diff
@@ -417,4 +419,139 @@ typedef enum {
+    {                                                                                                       \
+        SCENECMD_ID_ROOM_BEHAVIOR, currRoomUnk3,                                                           \
+            currRoomUnk2 | _SHIFTL(currRoomUnk5, 8, 1) | _SHIFTL(msgCtxunk12044, 10, 1) |                   \
+                _SHIFTL(enablePosLights, 11, 1) | _SHIFTL(kankyoContextUnkE2, 12, 1)                        \
+    }
+
+#define SCENECMD_UNUSED_9()                   \
+    { SCENECMD_ID_UNUSED_9, 0, CMD_W(0) }
+
+#define SCENECMD_MESH(meshHeader)                    \
+    { SCENECMD_ID_MESH, 0, CMD_PTR(meshHeader) }
+
+#define SCENECMD_OBJECT_LIST(objectNum, objectList)                 \
+    { SCENECMD_ID_OBJECT_LIST, objectNum, CMD_PTR(objectList) }
+
+#define SCENECMD_POS_LIGHT_LIST(unusedLightNum, unusedLightList)             \
+    { SCENECMD_ID_POS_LIGHT_LIST, unusedLightNum, CMD_PTR(unusedLightList) } 
+
+#define SCENECMD_PATH_LIST(pathList)                    \
+    { SCENECMD_ID_PATH_LIST, 0, CMD_PTR(pathList) }
+
+#define SCENECMD_TRANSI_ACTOR_LIST(transiNum, transiList)             \
+    { SCENECMD_ID_TRANSI_ACTOR_LIST, transiNum, CMD_PTR(transiList) } 
+
+#define SCENECMD_LIGHT_LIST(lightNum, lightList)                 \
+    { SCENECMD_ID_LIGHT_LIST, lightNum, CMD_PTR(lightList) }
+
+#define SCENECMD_TIME_SETTINGS(hour, min, speed)                        \
+    { SCENECMD_ID_TIME_SETTINGS, 0, CMD_BBBB(hour, min, speed, 0) }
+
+#define SCENECMD_SKYBOX_SETTINGS(externalTextureFileID, skyboxId, weather, lightMode)                     \
+    { SCENECMD_ID_SKYBOX_SETTINGS, externalTextureFileID, CMD_BBBB(skyboxId, weather, lightMode, 0) }
+
+#define SCENECMD_SKYBOX_DISABLES(disableSky, disableSunMoon)                           \
+    { SCENECMD_ID_SKYBOX_DISABLES, 0, CMD_BBBB(disableSky, disableSunMoon, 0, 0) }
+
+#define SCENECMD_EXIT_LIST(exitList)                    \
+    { SCENECMD_ID_EXIT_LIST, 0, CMD_PTR(exitList) }
+
+#define SCENECMD_END()                   \
+    { SCENECMD_ID_END, 0, CMD_W(0) }
+
+#define SCENECMD_SOUND_SETTINGS(audioSessionId, nighttimeSfx, bgmId)                    \
+    { SCENECMD_ID_SOUND_SETTINGS, audioSessionId, CMD_BBBB(0, 0, nighttimeSfx, bgmId) }
+
+#define SCENECMD_ECHO_SETTINGS(echo)                              \
+    { SCENECMD_ID_ECHO_SETTINGS, 0, CMD_BBBB(0, 0, 0, echo) }
+
+#define SCENECMD_CUTSCENE_LIST(cutsceneCount, cutsceneList)                 \
+    { SCENECMD_ID_CUTSCENE_LIST, cutsceneCount, CMD_PTR(cutsceneList) }
+
+#define SCENECMD_ALTERNATE_HEADER_LIST(alternateHeaderList)                    \
+    { SCENECMD_ID_ALTERNATE_HEADER_LIST, 0, CMD_PTR(alternateHeaderList) }
+
+// OoT
+#define SCENECMD_MISC_SETTINGS(camMode, worldMapLocation)               \
+    { SCENECMD_ID_MISC_SETTINGS, camMode, CMD_W(worldMapLocation) }
+// MM
+#define SCENECMD_WORLD_MAP_VISITED()                   \
```

Comment:
thinking we should leave out the MM specific commands in OoT. In a modding context if someone would like to back port some of the new commands they can add it themselves. But as far as OoT goes, they arent relevant here

Reply:
yeah, those were meant to only in the mm repo